packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -0,0 +1,235 @@
+        ) : (
+            <Localize i18n_default_text='Slippage happens when the asset price changes by the time it reaches our servers.' />
+        ),
+    [TERM.STOP_OUT_LEVEL]: (
+        <Localize i18n_default_text='Your trade will be closed automatically at the nearest available asset price when your loss reaches a certain percentage of your stake, but your loss never exceeds your stake. This percentage depends on the chosen underlying asset and the Multiplier.' />
+    ),
+    [TERM.STOP_LOSS]: (
+        <Localize i18n_default_text='If you select this feature, your trade will be closed automatically at the nearest available asset price when your loss reaches or exceeds the stop loss amount. Your loss may be more than the amount you entered depending on the market price at closing.' />
+    ),
+    [TERM.STRIKE_PRICE]: (
+        <React.Fragment>
+            <Text className='definition__paragraph'>
+                <Localize i18n_default_text='You must select the strike price before entering the contract.' />
+            </Text>
+            <Text className='definition__paragraph'>
+                <Localize i18n_default_text='If you select “Call”, you’ll earn a payout if the final price is above the strike price at expiry. Otherwise, you won’t receive a payout.' />
+            </Text>
+            <Text>
+                <Localize i18n_default_text='If you select ‘Put”, you’ll earn a payout if the final price is below the strike price at expiry. Otherwise, you won’t receive a payout.' />
+            </Text>
+        </React.Fragment>
+    ),
+    [TERM.TAKE_PROFIT]: (
+        <Localize i18n_default_text='If you select this feature, your trade will be closed automatically at the nearest available asset price when your profit reaches or exceeds the take profit amount. Your profit may be more than the amount you entered depending on the market price at closing.' />
+    ),
+};
+
+export const getTermDefinition = ({ contract_type, term }: { contract_type: string; term: string }) => {
+    const result = DEFINITION[term];
+    if (typeof result === 'function') return result(contract_type);
+    return result ?? '';
+};
+
+export const parseContractDescription = (
+    content: {
+        type: string;
+        text: JSX.Element | string;
+    }[]
+) =>
+    content.map(({ type, text }, index) => {
+        if (type === 'heading' && typeof text !== 'string')
+            return (
+                <Text key={text.props.i18n_default_text} bold size='md' className='description__heading'>
+                    {text}
+                </Text>
+            );
+        if ((type === 'paragraph' || type === 'general') && typeof text !== 'string')
+            return (
+                <Text
+                    as='p'
+                    key={text.props.i18n_default_text}
+                    size='sm'
+                    className={`description__${type === 'paragraph' ? 'paragraph' : 'general'}`}
```

Comment:
```suggestion
                    className={`description__${type}`}
```
here, type can be either 'paragraph' or 'general', right? )

Reply:
Oh, good idea, didn't notice it, thank you)